runtime/lib/formatter/PropelObjectFormatter.php

```diff
@@ -39,16 +39,21 @@ public function format(PDOStatement $stmt)
             $pks = array();
             $objectsByPks = array();
             while ($row = $stmt->fetch(PDO::FETCH_NUM)) {
-                $object = $this->getAllObjectsFromRow($row);
-                $pk = $object->getPrimaryKey();
 
                 if (false === Propel::isInstancePoolingEnabled()) {
+
+                    list($object, $col) = call_user_func(array($this->peer, 'populateObject'), $row);
```

Comment:
Hey @staabm ,

I actually copied this line from the method "getAllObjectsFromRow". Didn't want to change anything else as I don't completely understand yet the framework.